openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -68,6 +68,8 @@ class AccountView(APIView):
 
                 * goals: null or textual representation of goals
 
+                * bio: null or textural representation of user biographical information ("about me")
```

Comment:
Good questions-- @explorerleslie what do you think? Should we do something to escape or expunge HTML?

I think there is essentially no size limit.